### DISCUSSION TITLE
FLAC tags in freaccmd vs freac GUI

### DISCUSSION THREAD

**Discussion by Orionn2000** [enzo1982/freac#515] [Help & Support]
I have a batch script that I used previously to rip CDs to FLAC using freaccmd, then encode those to AAC & MP3. Now I have a new PC  (with new version of freac) and the script is behaving differently.

When I rip the tracks to FLAC using freaccmd, everything looks fine, but when I encode them to AAC/MP3 it thinks the track number is 1 for every track. If I rip the same CD in freac GUI it works fine. I have tried comparing the FLAC files from freaccmd and freac gui and I can't see any difference.

Can anyone suggest why the freaccmd tags look fine, but then all become track 1 when encoded from flac to aac/mp3?

Here are the commands I'm using:
`mkdir "%UserProfile%\Desktop\CDRips\temp"
"C:\Program Files\freac\freaccmd.exe" --cddb -e flac -cd 0 -t all -d "%UserProfile%\Desktop\CDRips" -p "temp\flac\<artist>\<album>\<track> - <title>"
for /R %UserProfile%\Desktop\CDRips\temp %f in (*.flac) do @"C:\Program Files\freac\freaccmd.exe" -e fdkaac -mp4 "%f" --cddb -p "Desktop\CDRips\AAC\<artist>\<album>\<track> - <title>"
for /R %UserProfile%\Desktop\CDRips\temp %f in (*.flac) do @"C:\Program Files\freac\freaccmd.exe" -e lame "%f" --cddb -p "Desktop\CDRips\MP3\<artist>\<album>\<track> - <title>"
robocopy %UserProfile%\Desktop\CDRips\temp\flac %UserProfile%\Desktop\CDRips\flac\ /E /IS /MOVE
rmdir %UserProfile%\Desktop\CDRips\temp /S /Q`

**Comment by Lee-Carre** [comment]
Have you tried saving your desired configuration as a profile within Fre:AC (GUI), and then telling the CLI binary to use that profile, instead of having it all within your script (unless there's good reason you want it that way?)?

Abstracting the configuration parameters away to the configuration file, would keep the script cleaner and more robust.
